shell/platform/darwin/macos/framework/Headers/FlutterEngine.h

```diff
@@ -76,10 +79,10 @@ FLUTTER_DARWIN_EXPORT
 - (BOOL)runWithEntrypoint:(nullable NSString*)entrypoint;
 
 /**
- * The default `FlutterViewController` associated with this engine, if any.
+ * The default `FlutterViewController` of this engine, if any.
  *
- * The default view always has ID kFlutterDefaultViewId, and is the view
- * operated by the APIs that do not have a view ID specified.
+ * The default view is the first view added to the engine, always has ID
+ * kFlutterDefaultViewId, and is used by legacy APIs that assume a single view.
```

Comment:
nit: seems odd to talk about the "first" view here since this can only have one (for now)?